app/components/ClinicUserBlock.js

```diff
@@ -42,22 +43,27 @@ class ClinicUserBlock extends React.Component {
   };
 
   render() {
+    var { allUsers, isUploadInProgress, memberships, targetId } = this.props;
+    var isCustodialAccount = _.has(_.get(memberships, [targetId, 'permissions']), 'custodian');
     var editClasses = cx({
       [styles.edit]: true,
-      [styles.disabled]: this.props.isUploadInProgress
+      [styles.disabled]: isUploadInProgress || !isCustodialAccount
```

Comment:
It's not - I was iterating through possible UX solutions and one was to simply disable the edit link instead of remove it (so there's not a UI disparity between having permissions and not, just the interaction would be different). I can remove that check.